base/number.jl

```diff
@@ -103,6 +103,9 @@ function getindex(x::Number, I::Integer...)
     @boundscheck all(isone, I) || throw(BoundsError())
     x
 end
+get(x::Number, i::Integer, default) = i == 1 ? x : default
+get(f::Callable, x::Number, i::Integer) = i == 1 ? x : f()
```

Comment:
Should this also have a method accepting `Tuple` as keys, since we do allow indexing numbers with multiple indices?

Reply:
Multiple or none. Can do, I wasn't sure how big to make this trivial thing...